### ISSUE TITLE
Feature request: multiple gateways

### ISSUE THREAD

**Issue by adrianmare** [rogierhofboer/com.ikea.tradfri.gateway#45]
Hello,

Could you add a feature for supporting multiple Tradfri gateways?

Thank you.

**Comment by json81** [comment]
I agree that this would be a nice feature. If you know how to clone an app it is easy to modify it and install a second app for use with another IKEA GW.

**Comment by MrMagu74** [comment]
This would be great, since I have 100+ items, and don't want to add them directly to Homey since I will miss firmware updates from Ikea

**Comment by bjornand1** [comment]
I second that! I have been trying a workaround to install two versions of the app, but for some reason, the second instance won't discover any devices connected to the second Gateway.

**Comment by Gizmopower1** [comment]
Yes. Would be great if possible
